ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -378,6 +378,13 @@ def pytest_addoption(parser):
         default=False,
         help=("Skips the RPM and go version collection for every pod for session"),
     )
+    parser.addoption(
```

Comment:
I don't think we need to have extra CLI argument for this option as I don't see this is something heavily used for what we need to have it.  This just can be easily populated by custom or extra config file (you can create such in `conf/ocsci/disable_high_recovery_during_rebalance.yaml`) for disabling this which can be passed via --ocsci-conf if needed.  And just have default value set in default config.